### PR TITLE
descriptives: add outlier labels option for box plots

### PR DESCRIPTION
**Pull request by raviselker**
* Added the following dependencies: 'ggrepel', 'tibble', 'dplyr', and 'magrittr'

### CODE REVIEW

File: R/descriptives.b.R [hunk 1]
```diff
@@ -1128,6 +1129,13 @@ descriptivesClass <- R6::R6Class(
                 else
                     x <- names$s1
 
+                if (self$options$box && self$options$boxLabelOutliers) {
+                    data <- data %>%
+                        tibble::rownames_to_column('.ROWNAMES') %>%
```
Comment (jonathon-love) [inline R/descriptives.b.R]: i appreciate `tibble` exists in the bowels of our dependencies anyway ... but i totally resent the hadleyverse ... can we drop this explicit dependency?

Comment (raviselker) [reply R/descriptives.b.R]: As in, don't use tibble, dplyr, and magrittr? Because these things make plotting a lot easier tbh

Comment (jonathon-love) [reply R/descriptives.b.R]: yeah, i'm only proposing to drop `tibble`

Comment (raviselker) [reply R/descriptives.b.R]: Just drop the `tibble` dependency, or the complete use?

Comment (jonathon-love) [reply R/descriptives.b.R]: so if we changed this:

```
data <- data %>%
    tibble::rownames_to_column('.ROWNAMES') %>%
```

to

```
data$.ROWNAMES <- rownames(data)
data <- data %>%
```

then we can ditch the `tibble` import

Comment (raviselker) [reply R/descriptives.b.R]: ok, I removed the dependency

**Pull request merged**